Layout documentation

joukevandermaas opened the pull request:
Preview: http://awesan.com/ember-paper/#/layout/introduction

DanChadwick commented:
Not using `{{paper-api}}` kinda defeats the purpose of letting us easily change the markup site-wide, no?

miguelcobain commented on tests/dummy/app/templates/layout/introduction.hbs:
```diff
@@ -0,0 +1,265 @@
+  </p>
+
+  <p>
+    The classes consist of a <em>attribute</em> and optional <em>parameters</em>, separated by
```
change to "an attribute".

miguelcobain commented on tests/dummy/app/templates/layout/introduction.hbs:
```diff
@@ -0,0 +1,265 @@
+    <div class="flex box-three"></div>
+  </div>
+  {{! END-SNIPPET }}
+  {{#card.content}}
```
Can we put the controls here? It gets hard to see the changes we're making because the controls are too far down (I'm on a small laptop).

miguelcobain commented on tests/dummy/app/templates/layout/introduction.hbs:
```diff
@@ -0,0 +1,265 @@
+    <div class="flex-33 flex-md-{{box1Height}} box-one"></div>
+    <div class="flex-33 layout-{{direction}} layout-md-row box-two">
+      <div class="flex-20 flex-md-10 hide-lg box-two-one"></div>
+      <div class="flex-30 {{if hideBox 'hide-md'}} flex-md-25 box-two-two"></div>
```
we're using double quotes everywhere in hbs files

miguelcobain commented:
This looks *very* good. Thanks you so much.

There is a problem that I'm not very sure how to solve: the page doesn't look good on mobile phones. :/

We can try to make responsive tables.

joukevandermaas commented:
@DanChadwick I couldn't get it working with all the various tables (like the one with two 'main' columns on on the intro page). Maybe I'll need to extend the paper API component first.

@miguelcobain It looks like the container element for the docs pages has a fixed width or something, I think that's the main problem. It seems to also happen on the general intro page and for some of the demos.

joukevandermaas commented:
@miguelcobain I cleaned up a bit, and renamed 'Demos' to 'Components' on @DanChadwick's suggestion. I don't think the responsiveness issues are really core to this PR (they were there already, just more obvious now), so I'd like to get this merged and attack that problem in a new PR if you agree.
